packages/jest-cli/src/reporters/VerboseReporter.js

```diff
@@ -50,14 +50,13 @@ class VerboseReporter extends DefaultReporter {
     results: AggregatedResult,
   ) {
     this._clearWaitingOn(config);
-    this._printTestFileHeaderAndFailures(config, testResult);
     if (!testResult.testExecError) {
       this._logTestResults(testResult.testResults);
     }
     this._printWaitingOn(results, config);
+    this._printTestFileHeaderAndFailures(config, testResult);
```

Comment:
why did you move this? It would print "Running XX tests" and then "PASS test.js" instead of the other way around like it was previously.